### DISCUSSION TITLE
📊 Mode expert > Ping domaine

### DISCUSSION THREAD

**Discussion by adriens** [lschaeffer313/domaine-nc-mobile#107] [Polls]
# :grey_question: Contexte

Il pourrait être utile de constater, directement depuis l'app les perfs du domaine, cela peut être fait avec le package [Ping](https://pub.dev/packages/dart_ping)

**Comment by lschaeffer313** [comment]
Fonctionnalité d'un terminal mais dans l'app mobile 📱 
Je valide ✅

**Reply by adriens** [reply]
Exactement, tu m'as capté :100: ... si on délire à bloc on pourrait faire un style terminal genre :nerd_face: 
![image](https://user-images.githubusercontent.com/5235127/210322827-0a60d6d3-a166-42fb-8aa2-31f29bb2a16d.png)
